OpenSim/Common/Test/testC3DFileAdapter.cpp

```diff
@@ -31,75 +31,132 @@
 #include <thread>
 #include <cmath>
 
-template<typename> class shrik;
-
-void compare_tables(const OpenSim::TimeSeriesTableVec3& table1,
-                    const OpenSim::TimeSeriesTableVec3& table2) {
+template<typename ETY = SimTK::Real>
+void compare_tables(const OpenSim::TimeSeriesTable_<ETY>& table1,
+            const OpenSim::TimeSeriesTable_<ETY>& table2,
+            const double tolerance = SimTK::SignificantReal,
+            const int nFramesToSkip = 1) {
```

Comment:
This method is called `compare_tables` but skips frames, so two tables may differ even if they pass through `compare_tables`. Before this method returns, I suggest `cout`ing the number of frames that were checked (e.g., "10 of 100 rows were tested and are equal").